back/app/services/participants_service.rb

```diff
@@ -193,15 +191,4 @@ def filter_engaging_activities(activities_scope)
     end
     output
   end
-
-  # Adds a `score` field to the results, indicating the engagement score for the activity
-  def with_engagement_scores(activities_scope)
```

Comment:
This was not used for anything so I removed it, seems really ancient code that Koen added 6 years ago